src/components/organisms/AdminVenueView/components/EventsView/EventsView.scss

```diff
@@ -0,0 +1,106 @@
+@import "scss/constants.scss";
+
+.EventsView {
+  background-color: #000;
+  position: relative;
+  height: 75%;
+  width: 85%;
+  margin: 75px 75px 10px;
+  border-radius: $border-radius--xl;
+  box-shadow: 0px 0px 15px -4px #000000;
+  overflow: scroll;
+
+  &__title {
+    margin: 20px;
+    font-weight: 500;
+    font-size: 20px;
+  }
+
+  &__content > .TimingEvent:nth-of-type(odd) {
+    background-color: $secondary--dark;
+  }
+
+  .TimingEvent {
+    display: flex;
+    width: 100%;
+    color: $concrete;
+    padding: 20px;
+
+    &__time {
+      position: relative;
+      display: inline;
+      float: left;
+      width: 15%;
+
+      p {
+        margin: 0px;
+      }
+
+      &-start {
+        font-weight: bold;
+        color: white;
+        margin-top: 20px;
+      }
+    }
+
+    &__details {
+      &-name {
+        font-weight: bold;
+        color: white;
+        font-size: 18px;
+      }
+
+      &-host {
+        font-weight: 600;
+      }
+
+      &-room {
+        span {
+          font-weight: bold;
+        }
+
+        font-size: 13px;
+      }
+
+      p {
+        margin: 0px;
+      }
+
+      position: relative;
+      display: inline;
+      width: 75%;
+    }
+
+    &__edit-button {
+      outline: none;
+      border: none;
+      width: $submit-button-diameter;
+      height: $submit-button-diameter;
+      border-radius: $border-radius--max;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin-left: $spacing--md;
+      background-color: $primary;
+    }
+  }
+
+  &__no-events {
+    width: 300px;
```

Comment:
I did. It looks good on most sizes, except very large screens, where the create events button isn't aligned with the events container because the padding doesn't work that way. I can fix that issue but I don't think it is a priority because on most laptop sizes it looks okay.

<img width="1154" alt="Screen Shot 2021-08-02 at 3 31 30 PM" src="https://user-images.githubusercontent.com/51083763/127878270-8a4b7ac3-a9de-42c4-848a-2b23cc852977.png">

![Screen Shot 2021-08-02 at 3 32 40 PM](https://user-images.githubusercontent.com/51083763/127878411-ba8ec1f5-9f90-46b2-8acb-af33d146b003.png)

Reply:
I took a lot of the values from the styleguide https://github.com/sparkletown/sparkle-styling

Reply:
> except very large screens, where the create events button isn't aligned with the events container because the padding doesn't work that way.

How are they aligned? They should be in a flex column, and that column should have a single padding value, you are probably setting a separate padding value for both elements and that's why they are different